src/content/topics/home/organizing-flags/environments.mdx

```diff
@@ -128,11 +132,12 @@ To reset an SDK or mobile key:
 
 1. Navigate to the the **Account settings** page.
 2. Click the **Projects** tab.
-3. Click the **overflow menu** next to your environment.
-4. Select "Edit environment." An "Edit environment" panel appears.
-5. In the "Keys" section click **Reset SDK key** or **Reset mobile key**.
-6. (Optional) If you are resetting an SDK key, choose how many hours to keep the current SDK key active for, up to 720 hours (30 days).
-6. Enter the name or key of your environment:
+3. In the **Projects** results click on a project name to navigate to that project.
```

Comment:
```suggestion
3. Navigate to the selected project by clicking the name of the project.
```
If I'm following correctly, this step is the same as what you're doing in environments.mdx/line 36. Is that right? If so, let's use the same language.